Patch to address:

ld: fatal: /builds/nrau/userland-build-test/components/x11/lib/libxkbcommon/libxkbcommon-1.0.1/xkbregistry.map: 1: expected version name or '{' in GNU version script: versions

--- a/xkbregistry.map	2020-10-08 09:37:02.070664432 -0700
+++ b/xkbregistry.map	2020-10-08 09:37:06.499753552 -0700
@@ -1,4 +1,3 @@
-/* versions are kept in sync with libxkbcommon.so */
 V_1.0.0 {
 global:
         rxkb_context_new;
